src/server/static.js

```diff
@@ -29,14 +28,31 @@ module.exports = bp => {
       })
     }
 
-    const requestPath = `/js/modules/${name}.js`
-    app.use(requestPath, (req, res) => {
+    const liteDir = path.join(module.root, module.settings.liteDir || 'bin/lite')
+    const liteViews = fs.existsSync(liteDir)
+      ? fs.readdirSync(liteDir).filter(b => b.endsWith('.js'))
+      : []
+
+    app.get([
+      `/js/modules/${shortName}`, // The full module view
+      `/js/modules/${name}.js`, // <<-- DEPRECATED: Will be removed shortly. Only use shortNames
```

Comment:
Do not forget to remove it if it's no more useful and if you are able to remove it now..